app/templates/sites/network-sites/network-site/buildings/new.hbs

```diff
@@ -0,0 +1,24 @@
+<h2>New Building</h2>
+
+{{#g-maps-container
+    lat=model.site.lat
+    lng=model.site.lng
+    zoom=16
+    buildingMarkers=buildingMarkers
```

Comment:
`buildingMarkers` is a computed property in the controller. Is there a way to avoid the computed prop? https://github.com/stevenwu/noovis_app2_ember2/blob/add-buildings/app/controllers/sites/network-sites/network-site/buildings/new.js